### PR TITLE
Use extension vendor for Psalm and Stan

### PR DESCRIPTION
**Pull request by malberts**
This means we have to `composer install` in the extension dir first.

When running the Make command on https://github.com/ProfessionalWiki/WikibaseExport/pull/33 I now also get the same error as the CI run:
Screenshot_20221111_101114

----

This isn't strictly necessary for PHPStan, since it is not (currently) held back due to MW pinning a dependency.

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
This change comes with a complication, though one that might not matter for this extension, at least at the moment.

Dependencies will end up being installed twice. This is annoying since then PhpStorm does not know which of the definitions to use. At least when the extension has production dependencies on libraries, which WikibaseExport currently does not.

Perhaps we can mark the vendor dir in the extension as excluded to solve the problem. 

TL;DR: maybe there would be a problem in some cases but we might be able to fix that, so whatever :)

**Comment by malberts** [comment]
If we do run into massive issues, then we can rather go the opposite direction and change CI to use Psalm in MW's vendor instead.

**Comment by JeroenDeDauw** [comment]
Looks like this change is actually causing troubles:

image

Strangely enough those errors only show up after fixing the original failure. That one is now fixed on https://github.com/ProfessionalWiki/WikibaseExport/pull/33
